### PR TITLE
DB-108: replace Mozilla Maintenance Service with Cliqz Service

### PR DESCRIPTION
**Pull request by alver-cliqz**
ServicesHelper.dll was replace with our build of this library, because need to change registry path in it.

All other text, related to this service, replaced to create a separate service, with similar functionality, as Mozilla's had.

### CODE REVIEW

**Comment by luciancor** [comment]
@maxim-cliqz can you please take a look on this PR?

File: mozilla-release/toolkit/components/maintenanceservice/bootstrapinstaller/maintenanceservice_installer.nsi [hunk 1]
```diff
@@ -45,16 +45,16 @@ Var BrandFullName
 ; And anyone that wants to run tests themselves should already have 
 ; this installed.
 !define FallbackKey \
-  "SOFTWARE\Mozilla\MaintenanceService\3932ecacee736d366d6436db0f55bce4"
+  "SOFTWARE\CLIQZ\MaintenanceService\3932ecacee736d366d6436db0f55bce4"
 
-!define CompanyName "Mozilla Corporation"
+!define CompanyName "CLIQZ GmbH"
```
Comment (maxim-cliqz) [inline mozilla-release/toolkit/components/maintenanceservice/bootstrapinstaller/maintenanceservice_installer.nsi]: I think the official name is "Cliqz GmbH"

**Comment by maxim-cliqz** [comment]
How about to create a Jira ticket to get rid of prebuilt dll and instead build it as needed?

**Comment by maxim-cliqz** [comment]
Looks OK

File: mozilla-release/toolkit/components/maintenanceservice/bootstrapinstaller/maintenanceservice_installer.nsi (hunk 1 above)
Comment (alver-cliqz) [inline mozilla-release/toolkit/components/maintenanceservice/bootstrapinstaller/maintenanceservice_installer.nsi]: Yes, right. Will change now.

**Pull request closed without merging**